packages/core/bootstrap/src/lib/rate-limit/index.ts

```diff
@@ -102,8 +102,8 @@ export const withRateLimit = (store: Store<RootState>): Middleware => async (exe
     participant_id: requestTypeId,
     experimental: 'true',
   }
-  const cost = Number(result.debug?.providerCost) || 1
-  metrics.rateLimitCreditsSpentTotal.labels(defaultLabels).inc(cost)
+  const cost = Number(result.debug?.providerCost)
```

Comment:
Can cost be fractional? Otherwise we should use `parseInt`

Reply:
It should never be fractional, but we could use `parseInt` just in case